src/index.html

```diff
@@ -22,6 +22,397 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo_logo"
```

Comment:
For the alt attribute we don't use _ or shortcuts as this is text for people with disabilities, so it should be readable
```suggestion
          alt="Moyo logo"
```